### ISSUE TITLE
can this work on a arduino nano ??

### ISSUE THREAD

**Issue by jrubiocl** [abratchik/HIDPowerDevice#34]
can this work on a arduino nano ??

regards

**Comment by RaduTek** [comment]
No, you need an Arduino with a built-in USB interface, because it uses HID. This library is meant for the Leonardo/Pro Micro, based on the Atmega 32u4 microcontroller.
